sqswatcher/sqswatcher.py

```diff
@@ -216,6 +235,10 @@ def main():
     region, sqsqueue, table_name, scheduler, cluster_user, proxy_config = getConfig()
     q = setupQueue(region, sqsqueue, proxy_config)
     t = setupDDBTable(region, table_name, proxy_config)
+    t.put_item(Item={
```

Comment:
This can be very dangerous in case the sqswatcher crashes and is restarted by the supervisord. In that case the current_cluster_size is set to 0 and the existing nodes will be never terminated.

Reply:
Read the value from the scheduler

Reply:
Yes, Maurizio and I discussed this over. We should write the current_value to the file or something, but sqswatcher crashing implies no new nodes can join, and the nodes can still terminate outside of CFNCluster ecosystem and the event is not read by the  master. If SQSWatcher crashes, there might be a lot of instability in the system anyways.